src/main/java/eu/dissco/core/digitalmediaprocessor/service/ProcessingService.java

```diff
@@ -111,59 +130,104 @@ private Set<DigitalMediaObjectEvent> removeDuplicatesInBatch(
     return uniqueSet;
   }
 
-  private void republishEvent(DigitalMediaObjectEvent event) {
+  private void republishEvent(DigitalMediaEvent event) {
     try {
-      kafkaService.republishDigitalMediaObject(event);
+      kafkaService.republishDigitalMedia(event);
     } catch (JsonProcessingException e) {
       log.error("Fatal exception, unable to republish message due to invalid json", e);
     }
   }
 
-  private ProcessResult processDigitalMedia(Set<DigitalMediaObjectEvent> events) {
+  private ProcessResult processDigitalMedia(Set<DigitalMediaEvent> events) {
     var currentMediaObjects = getCurrentDigitalMedia(
```

Comment:
elsewhere, we call these kinds of "existingObjects", maybe change current -> existing. That makes more sense to me

Reply:
I think of "existingXYZ" as the objects we pull from the database. When we compare an updated version of the object to its new version, then we have "currentDigitalMedia" and "digitalMedia". just my preference, I won't die on this hill

Reply:
Oeh I think we call it currentXXX everywhere, switch exisiting here to current. If we want to changes this to existing we should do it in all places (and create a separate ticket for it)